darwin/server/default.nix

```diff
@@ -1 +1,12 @@
-{ imports = [ ../common ]; }
+{ lib, pkgs, ... }:
+{
+  imports = [
+    ../common
+    ../../shared/server.nix
+  ];
+
+  environment.systemPackages = map lib.lowPrio [
+    # no config.programs.git.package option on darwin
+    pkgs.git
```

Comment:
This could be represented by ``config.programs.git.package or pkgs.git``

Reply:
https://github.com/nix-community/srvos/pull/542